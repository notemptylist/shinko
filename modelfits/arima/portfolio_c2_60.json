{
    "numlags": 400,
    "results": [
        {
            "mean": -0.007608476578040261,
            "order": [
                0,
                0,
                1
            ],
            "rmse": 1.7610478586384235,
            "tstamp": 1640145774.7479908
        },
        {
            "mean": -0.007608476578040261,
            "order": [
                6,
                1,
                1
            ],
            "rmse": 1.8895089060701544,
            "tstamp": 1640145774.7479908
        },
        {
            "mean": -0.007608476578040261,
            "order": [
                0,
                1,
                1
            ],
            "rmse": 2.7737221654119124,
            "tstamp": 1640145774.7479908
        },
        {
            "mean": -0.003166810047224998,
            "order": [
                5,
                0,
                1
            ],
            "rmse": 1.3403773075092107,
            "tstamp": 1640309675.6964543
        },
        {
            "mean": -0.003166810047224998,
            "order": [
                6,
                0,
                1
            ],
            "rmse": 1.340465255284863,
            "tstamp": 1640309675.6964543
        },
        {
            "mean": -0.003166810047224998,
            "order": [
                6,
                3,
                0
            ],
            "rmse": 2.9852519629071885,
            "tstamp": 1640309675.6964543
        }
    ],
    "stream": "portfolio_c2_60.json",
    "todo": [
        [
            0,
            0,
            0
        ],
        [
            0,
            0,
            3
        ],
        [
            0,
            1,
            0
        ],
        [
            0,
            1,
            3
        ],
        [
            0,
            3,
            0
        ],
        [
            0,
            3,
            1
        ],
        [
            0,
            3,
            3
        ],
        [
            1,
            0,
            0
        ],
        [
            1,
            0,
            1
        ],
        [
            1,
            0,
            3
        ],
        [
            1,
            1,
            0
        ],
        [
            1,
            1,
            1
        ],
        [
            1,
            1,
            3
        ],
        [
            1,
            3,
            0
        ],
        [
            1,
            3,
            1
        ],
        [
            1,
            3,
            3
        ],
        [
            2,
            0,
            0
        ],
        [
            2,
            0,
            1
        ],
        [
            2,
            0,
            3
        ],
        [
            2,
            1,
            0
        ],
        [
            2,
            1,
            1
        ],
        [
            2,
            1,
            3
        ],
        [
            2,
            3,
            0
        ],
        [
            2,
            3,
            1
        ],
        [
            2,
            3,
            3
        ],
        [
            3,
            0,
            0
        ],
        [
            3,
            0,
            1
        ],
        [
            3,
            0,
            3
        ],
        [
            3,
            1,
            0
        ],
        [
            3,
            1,
            1
        ],
        [
            3,
            1,
            3
        ],
        [
            3,
            3,
            0
        ],
        [
            3,
            3,
            1
        ],
        [
            3,
            3,
            3
        ],
        [
            4,
            0,
            0
        ],
        [
            4,
            0,
            1
        ],
        [
            4,
            0,
            3
        ],
        [
            4,
            1,
            0
        ],
        [
            4,
            1,
            1
        ],
        [
            4,
            1,
            3
        ],
        [
            4,
            3,
            0
        ],
        [
            4,
            3,
            1
        ],
        [
            4,
            3,
            3
        ],
        [
            5,
            0,
            0
        ],
        [
            5,
            0,
            3
        ],
        [
            5,
            1,
            0
        ],
        [
            5,
            1,
            1
        ],
        [
            5,
            1,
            3
        ],
        [
            5,
            3,
            0
        ],
        [
            5,
            3,
            1
        ],
        [
            5,
            3,
            3
        ],
        [
            6,
            0,
            0
        ],
        [
            6,
            0,
            3
        ],
        [
            6,
            1,
            0
        ],
        [
            6,
            1,
            3
        ],
        [
            6,
            3,
            1
        ],
        [
            6,
            3,
            3
        ]
    ],
    "version": "0.1"
}