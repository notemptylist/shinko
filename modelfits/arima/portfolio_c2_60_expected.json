{
    "numlags": 400,
    "results": [
        {
            "mean": -0.012724974301006223,
            "order": [
                3,
                0,
                3
            ],
            "rmse": 0.2889666633522978,
            "tstamp": 1640381609.939555
        },
        {
            "mean": -0.012724974301006223,
            "order": [
                5,
                3,
                1
            ],
            "rmse": 0.3872334043630757,
            "tstamp": 1640381609.939555
        },
        {
            "mean": -0.012724974301006223,
            "order": [
                2,
                3,
                3
            ],
            "rmse": 17135.10304464789,
            "tstamp": 1640381609.939555
        }
    ],
    "stream": "portfolio_c2_60_expected.json",
    "todo": [
        [
            0,
            0,
            0
        ],
        [
            0,
            0,
            1
        ],
        [
            0,
            0,
            3
        ],
        [
            0,
            1,
            0
        ],
        [
            0,
            1,
            1
        ],
        [
            0,
            1,
            3
        ],
        [
            0,
            3,
            0
        ],
        [
            0,
            3,
            1
        ],
        [
            0,
            3,
            3
        ],
        [
            1,
            0,
            0
        ],
        [
            1,
            0,
            1
        ],
        [
            1,
            0,
            3
        ],
        [
            1,
            1,
            0
        ],
        [
            1,
            1,
            1
        ],
        [
            1,
            1,
            3
        ],
        [
            1,
            3,
            0
        ],
        [
            1,
            3,
            1
        ],
        [
            1,
            3,
            3
        ],
        [
            2,
            0,
            0
        ],
        [
            2,
            0,
            1
        ],
        [
            2,
            0,
            3
        ],
        [
            2,
            1,
            0
        ],
        [
            2,
            1,
            1
        ],
        [
            2,
            1,
            3
        ],
        [
            2,
            3,
            0
        ],
        [
            2,
            3,
            1
        ],
        [
            3,
            0,
            0
        ],
        [
            3,
            0,
            1
        ],
        [
            3,
            1,
            0
        ],
        [
            3,
            1,
            1
        ],
        [
            3,
            1,
            3
        ],
        [
            3,
            3,
            0
        ],
        [
            3,
            3,
            1
        ],
        [
            3,
            3,
            3
        ],
        [
            4,
            0,
            0
        ],
        [
            4,
            0,
            1
        ],
        [
            4,
            0,
            3
        ],
        [
            4,
            1,
            0
        ],
        [
            4,
            1,
            1
        ],
        [
            4,
            1,
            3
        ],
        [
            4,
            3,
            0
        ],
        [
            4,
            3,
            1
        ],
        [
            4,
            3,
            3
        ],
        [
            5,
            0,
            0
        ],
        [
            5,
            0,
            1
        ],
        [
            5,
            0,
            3
        ],
        [
            5,
            1,
            0
        ],
        [
            5,
            1,
            1
        ],
        [
            5,
            1,
            3
        ],
        [
            5,
            3,
            0
        ],
        [
            5,
            3,
            3
        ],
        [
            6,
            0,
            0
        ],
        [
            6,
            0,
            1
        ],
        [
            6,
            0,
            3
        ],
        [
            6,
            1,
            0
        ],
        [
            6,
            1,
            1
        ],
        [
            6,
            1,
            3
        ],
        [
            6,
            3,
            0
        ],
        [
            6,
            3,
            1
        ],
        [
            6,
            3,
            3
        ]
    ],
    "version": "0.1"
}